### DISCUSSION TITLE
question about drush and performance

### DISCUSSION THREAD

**Discussion by tridoxx** [GetDKAN/dkan#3885] [Q&A]
hello everyone, I would like to know if when using drush, the ram that it will use to do the harvest tests is automatically taken from the machine's available or some additional change must be made, so that when doing the harvest, the machine no crash or harvest process.

Thank you for your answers

**Comment by janette** [comment]
@tridoxx Harvesting will use the resources available on your web server. Look at your php error log to get information on when and why it is crashing.
